synapse/config/registration.py

```diff
@@ -378,6 +378,9 @@ def generate_config_section(self, generate_secrets=False, **kwargs):
         # By default, any room aliases included in this list will be created
         # as a publicly joinable room when the first user registers for the
         # homeserver. This behaviour can be customised with the settings below.
+        # If the room already exists, make certain it is a publicly joinable
+        # room, i.e. 'Security & Privacy' settings for the room allow 'Anyone
+        # who knows the room's link, including guests' to access the room.
```

Comment:
This is very helpful, though I'm afraid Element is only one client in the Matrix ecosystem, and we try to be client-agnostic in Synapse.

The requirement for the room is to be publicly joinable (have a join rule of `public` set). Additionally one doesn't need to allow guest access to the room.

```suggestion
        # room. The join rule of the room must be set to 'public'.
```

While I'd like to make this as easy as possible for people to use, client-specific instructions are best suited to a client-specific guide.